### DISCUSSION TITLE
Ubuntu 9.10 theme

### DISCUSSION THREAD

**Discussion by yumnycheesecakes** [mate-desktop/mate-desktop#614] [Q&A]
since Mate is based on Gnome 2, is it possible to make Mate look like Ubuntu 9.10?

**Comment by lukefromdc** [comment]
The GTK theme would have to be ported to GTK3 (if it hasn't been already), like what I did with my UbuntuStudio_Legacy theme with a modded version of the circa 2008 UbuntuStudio theme as a starting point.
